app/models/user.rb

```diff
@@ -28,6 +27,16 @@ def total_items_purchased
   end
 
   def format_phone
-    self.phone = self.phone.gsub(/[^0-9]/, '') unless self.phone.nil? || self.phone.empty?
+    self.phone = self.phone.gsub(/[^0-9]/, '')
+  end
+
+  def self.find_or_create_by_auth(auth_data)
```

Comment:
Assignment Branch Condition size for find_or_create_by_auth is too high. [15.07/15]